hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/DefineGroupsLayout.java

```diff
@@ -463,12 +473,18 @@ private void removeGroupRow(final GroupRow groupRow) {
         private void validateMandatoryPercentage(final Object value) {
             if (value != null) {
                 final String message = i18n.getMessage("message.rollout.field.value.range", 0, 100);
+                RangeValidator rangeValidator;
                 if (value instanceof Float) {
-                    new FloatRangeValidator(message, 0F, 100F).validate(value);
+                    rangeValidator = new FloatRangeValidator(message, 0F, 100F);
+                }
+                else if (value instanceof Integer) {
+                    rangeValidator = new IntegerRangeValidator(message, 0, 100);
                 }
-                if (value instanceof Integer) {
-                    new IntegerRangeValidator(message, 0, 100).validate(value);
+                else {
+                    throw new Validator.InvalidValueException(i18n.getMessage("message.rollout.field.value.type"));
```

Comment:
I would use the _message.enter.number_ here as before, because the entered value is not empty and not a number in this case. And in case the value itself is empty (null), I would define the message like "Field can not be empty"

Reply:
Non-numeric values are handled somewhere else in the first place, showing the message "Could not convert value to Float". Thus, the Exception here is never thrown anyway (or only in unexpected cases). But to avoid introducing a new message string I'll just change this as you suggested. 
An empty field is already handled by throwing the EmptyValueException and showing the _message.enter.number_.